### PR TITLE
Fixes #18990 - Refactor subscriptions add/remove tables

### PR DESCRIPTION
**Pull request by johnpmitsch**
added links and made the subscription title bold, we also
use the same code in 4 places (AK add/remove subs and CH add/
remove subs), moved that to a single template.

### CODE REVIEW

**Comment by johnpmitsch** [comment]
screenshots:

## before ##
before

## after ##
after

File: engines/bastion_katello/app/assets/stylesheets/bastion_katello/host_subscriptions.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  tbody {
+    tr.row-selector-label {
+      td {
+        &.row-select {
+          border-right: 0;
+        }
+        &:last-of-type {
+          border-left: 0;
+          position: relative;
+          left: -30px;
```
Comment (johnpmitsch) [inline engines/bastion_katello/app/assets/stylesheets/bastion_katello/host_subscriptions.scss]: @ehelms I'm trying to use the $row-select-width variable from [here](https://github.com/Katello/bastion/pull/178/files) but  I get an error when I use it. Do I have to import something to use bastion SASS variables?

**Comment by jlsherrill** [comment]
[test]

File: engines/bastion_katello/app/assets/stylesheets/bastion_katello/host_subscriptions.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+table.table-bordered {
+  tbody {
+    tr.row-selector-label {
```
Comment (waldenraines) [inline engines/bastion_katello/app/assets/stylesheets/bastion_katello/host_subscriptions.scss]: Do you need all this specificity to override a class?  Generally you only use the amount of specificity needed to override something.  

If we removed `.table-bordered` here then none of these styles would apply any longer.  That example is somewhat contrived but you can see how fragile extra specificity can be.  I'd recommend removing `.table-bordered` at a minimum.

**Comment by johnpmitsch** [comment]
@waldenraines updated to use less specificity in the css

**Comment by johnpmitsch** [comment]
@ehelms @waldenraines updated to use bastion variable from https://github.com/Katello/bastion/pull/178/files

**Comment by waldenraines** [comment]
This is a nice improvement, thanks @johnpmitsch

**Comment by ehelms** [comment]
[test]

**Comment by ehelms** [comment]
@johnpmitsch I released Bastion 4.3.0, and restarted the tests for this.

**Comment by ehelms** [comment]
@johnpmitsch looks like you are good to go here

**Comment by johnpmitsch** [comment]
@ehelms @waldenraines thanks!

**Pull request merged**